src/Application.php

```diff
@@ -292,6 +293,11 @@ protected function loadConfig()
             define('CRAFT_TRANSLATIONS_PATH', rtrim($config['craft_translations_path'], '/').'/');
         }
 
+        if (isset($config['dotenv_path'])) {
```

Comment:
I'd like to see a default `dotenv_path` set here in case the user does not have a `.craft-cli.php` config file.

```
$dotenvPath = isset($config['dotenv_path']) ? $config['dotenv_path'] : getcwd();
```

This is also where you should add the file exists check so we can have a default `dotenv_path`:

```
if (file_exists($dotenvPath.'/.env')) {
    $dotenv = new Dotenv($config['dotenv_path']);
    $dotenv->load();
}
```